frontend/components/Auction.vue

```diff
@@ -1,17 +1,21 @@
 <template>
     <TextBlock :title="`Auction #${auctionId}`">
-        <Alert v-if="auctionError" class="my-3" :message="auctionError" type="error" />
+        <div class="my-3">
+            <Alert v-if="auctionError && auctionError.showBanner" :message="auctionError.error" type="error" />
+        </div>
         <div v-if="auction">
-            <RestartBlock
-                v-if="auctionError === 'This auction is inactive and must be restarted'"
+            <AuctionRestartPanel
+                :disabled="!walletAddress"
+                :auction-is-active="auction.isActive"
```

Comment:
I suppose that you don't need to pass `isActive` as a prop since the panel shouldn't be shown if the action is active, right? (in contrast to other panels, so maybe it's not that obvious)
```suggestion
                v-if="auction.isActive"
```